boards/arm/mimxrt1060_evk/doc/mimxrt1060_evk.rst

```diff
@@ -0,0 +1,195 @@
+.. _mimxrt1060_evk:
+
+NXP MIMXRT1060-EVK
+##################
+
+Overview
+********
+
+The i.MX RT1060 is the latest addition to the industry's first crossover
+processor series and expands the i.MX RT series to three scalable families.
+The i.MX RT1060 doubles the On-Chip SRAM to 1MB while keeping pin-to-pin
+compatibility with i.MX RT1050. This new series introduces additional features
+ideal for real-time applications such as High-Speed GPIO, CAN-FD, and
+synchronous parallel NAND/NOR/PSRAM controller. The i.MX RT1060 runs on the
+Arm® Cortex-M7® core at 600 MHz.
+
+.. image:: mimxrt1060_evk.jpg
+   :width: 720px
+   :align: center
+   :alt: MIMXRT1060-EVK
+
+Hardware
+********
+
+- MIMXRT1062DVL6A MCU (600 MHz, 1024 KB on-chip memory)
+
+- Memory
+
+  - 256 Mbit SDRAM
+  - 64 Mbit QSPI Flash
+  - 512 Mbit Hyper Flash
+  - TF socket for SD card
+
+- Display
+
+  - LCD connector
+
+- Ethernet
+
+  - 10/100 Mbit/s Ethernet PHY
+
+- USB
+
+  - USB 2.0 OTG connector
+  - USB 2.0 host connector
+
+- Audio
+
+  - 3.5 mm audio stereo headphone jack
+  - Board-mounted microphone
+  - Left and right speaker out connectors
+
+- Power
+
+  - 5 V DC jack
+
+- Debug
+
+  - JTAG 20-pin connector
+  - OpenSDA with DAPLink
+
+- Sensor
+
+  - FXOS8700CQ 6-axis e-compass
+  - CMOS camera sensor interface
+
+- Expansion port
+
+  - Arduino interface
+
+- CAN bus connector
+
+For more information about the MIMXRT1060 SoC and MIMXRT1060-EVK board, see
+these references:
+
+- `i.MX RT1060 Website`_
+- `i.MX RT1060 Datasheet`_
+- `i.MX RT1060 Reference Manual`_
+- `MIMXRT1060-EVK Website`_
+- `MIMXRT1060-EVK User Guide`_
+- `MIMXRT1060-EVK Schematics`_
+
+Supported Features
+==================
+
+The mimxrt1060_evk board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
```

Comment:
delete the blank line and indenting:

    The default configuration can be found in the defconfig file:
    ``boards/arm/mimxrt1060_evk/mimxrt1060_evk_defconfig``.

Reply:
ok